documentation/anonymization.markdown

```diff
@@ -4,20 +4,22 @@ layout: default
 canonical: "/puppetdb/latest/anonymization.html"
 ---
 
-The export tool has the ability to anonymize the PuppetDB archive before returning the archive. This is particularly useful when sharing PuppetDB data that contains sensitive items.
+The export tool has the ability to anonymize the PuppetDB archive before
+returning the archive. This is particularly useful when sharing PuppetDB data
+that contains sensitive items.
 
 Using the `export` command
 -----
 
-To create an anonymized PuppetDB archive directly, use the Puppet `db` subcommand
-from any node with puppet-client-tools installed:
+To create an anonymized PuppetDB archive directly, use the Puppet `db`
+subcommand from any node with puppet-client-tools installed:
 
-    $ puppet db export --outfile my-puppetdb-export.tar.gz --anonymization moderate
```

Comment:
I count 66 cases where we use $ in front of a cli command, and you've changed 5 of them in this PR, and added a whole bunch of bare ones as new ones. If we're going to change a convention, we should do it to the lot, or not at all.